src/Components/navbar/navbar.jsx

```diff
@@ -20,11 +23,11 @@ function Navbar() {
                         <div class="icon-badge">
```

Comment:
Also you are using anchor tags for home login and sign use 'Link' or 'Navlink' as there's no index.html or login.html file.